packages/react/src/components/Button/ButtonThemes.ts

```diff
@@ -157,7 +157,6 @@ export function primaryStyles(theme: ITheme): IButtonStyles {
   return {
     root: {
       backgroundColor: s.primaryButtonBackground,
-      border: `1px solid ${s.primaryButtonBackground}`,
```

Comment:
Is this not removing the borders from all primary buttons and not only from `SplitButtons`?

Reply:
Oh true, I think I have to add a selector instead. Thanks for pointing it out.